src/OpenTelemetry.ClrProfiler.Managed/Instrumentations/MongoDb/MongoDbIntegration.cs

```diff
@@ -15,9 +16,11 @@ namespace OpenTelemetry.ClrProfiler.Managed.Instrumentations.MongoDb
     /// <summary>
     /// Tracing integration for MongoDB.Driver.Core.
     /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I am curious why is it needed

Reply:
They hide it from the editor/IDE so it doesn't offer this on the auto-complete and related IDE stuff - likely to avoid someone directly referencing the instrumentation code by accident, one needs to be intentional about it.